changelog/fragments/fix-csv-role-names.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      Fixed incorrect (cluster) role name assignments in generated CSVs
+      [#3600](https://github.com/operator-framework/operator-sdk/issues/3600).
```

Comment:
We do not add the number of the issue. We just add the link of the PR. I think it can bring confusing since it is not the standard. 
```suggestion

```

Reply:
Its helpful to link the issue. In fact I was thinking about making a modification to the changelog generator to add an optional `issue_link` field. Perhaps some wrapping words to make what this link is obvious.

Reply:
If we just add #number users will expect it to be the PR. Also, in this case it needs to be fixed manually in the release since the changelog is gen automatically. 

I am ok with we start to add the number the issue if it is identified as the gen changelog be changed to accommodate this new info.